lib/commands/arm/powerbiembedded/powerbiembedded._js

```
@@ -0,0 +1,349 @@
+    .option('-n --name <name>', $('Name of workspace collection'))
+    .option('-s --subscription [subscription]', $('The subscription identifier'))
+    .execute(function (resourceGroup, name, options, _) {
+      /** Validate parameters */
+      if (!resourceGroup) {
+        return cli.missingArgument('resourceGroup');
+      }
+      if (!name) {
+        return cli.missingArgument('name');
+      }
+      
+      /** Create client */
+      var subscription = profile.current.getSubscription(options.subscription);
+      var client = utils.createPowerbiManagementClient(subscription);
+      
+      /** Invoke client method */
+      var progress = cli.interaction.progress($('Getting workspace collection access keys...'));
+      var accessKeys;
+      try {
+        accessKeys = client.workspaceCollections.getAccessKeys(resourceGroup, name);
+      } finally {
+        progress.end();
+      }
+
+      /** Output result */
+      log.info('AccessKeys:');
+      log.info('Key1: ' + accessKeys.key1);
+      log.info('Key2: ' + accessKeys.key2);
+    });
+  
+  /**
+   * Regenerage access key
+   */
+  powerbiembeddedcli.command('regenerate-key <resourceGroup> <name> <keyName>')
+    .description($('Get access keys for a workspace collection'))
+    .option('-g --resource-group <resourceGroup>', $('Name of the resource group'))    
+    .option('-n --name <name>', $('Name of workspace collection'))
+    .option('-k --keyName <keyName>', $('Name of key you would like to regenerate. Accepted values: "key1" (Primary) or "key2" (Secondary)'))
+    .option('-s --subscription [subscription]', $('The subscription identifier'))
+    .execute(function (resourceGroup, name, keyName, options, _) {
+      /** Validate parameters */
+      if (!resourceGroup) {
+        return cli.missingArgument('resourceGroup');
+      }
+      if (!name) {
+        return cli.missingArgument('name');
+      }
+      if (!keyName) {
+        return cli.missingArgument('keyName');
+      }
```

Yea, I will do that. Our api would catch it but it's better to fail faster i think